Sprint 1 PD/app.py

```diff
@@ -0,0 +1,62 @@
+from flask import Flask, request, jsonify
+from flask_sqlalchemy import SQLAlchemy
+from flask_cors import CORS
+
+app = Flask(__name__)
+CORS(app)
+
+# Configuration for the SQLite database
+app.config['SQLALCHEMY_DATABASE_URI'] = 'mysql+pymysql://admin:pfwfall2024@donnadb.cvaai4e089sw.us-east-2.rds.amazonaws.com:3306/appdetail'
```

Comment:
put user name, password, and the link to db in the environment variables in ".env" file.